# request params
sensor_msgs/Image color_image
string prompt
---
# response params
string[] labels
std_msgs/Float32MultiArray boxes
float32[] scores
sensor_msgs/Image annotated_frame
sensor_msgs/Image segmask